SimCalorimetry/Configuration/python/SimCalorimetry_EventContent_cff.py

```diff
@@ -44,6 +44,9 @@
 run2_common.toModify( SimCalorimetryFEVTDEBUG.outputCommands, func=lambda outputCommands: outputCommands.append('keep *_simHcalUnsuppressedDigis_*_*') )
 run2_common.toModify( SimCalorimetryRAW.outputCommands, func=lambda outputCommands: outputCommands.append('keep *_simHcalUnsuppressedDigis_*_*') )
 
+from Configuration.Eras.Modifier_phase2_ecal_devel_cff import phase2_ecal_devel
+phase2_ecal_devel.toModify(SimCalorimetryFEVTDEBUG.outputCommands, func=lambda outputCommands: outputCommands.append('keep *_ecal*_*_*') )
```

Comment:
Wild cards in the output commands should better get avoided, in order to avoid the proliferation of unwanted collections in the output files. 

On the other hand I understand from https://github.com/cms-sw/cmssw/pull/36748#discussion_r788572802 that this is exactly what you want for it. Still, is the list of collections to be saved really unknown a priori? Cannot you add them one by one, even if you get a long list with it? Please check, and update with the explicit products if you think that you can also satisfy your needs as such.

Reply:
Sine this is a development modifier that should not be running in production it seemed to make the development process easier if all products from ECAL related modules (at least those having ecal in the name) are kept by default. It shows what is produced (and eventually should not be because it is e.g. for Phase 1) and what is missing. Since this WF is for ongoing development we do not know a priori what collections will be produced.